### ISSUE TITLE
/usr/bin/steamcmd bash shebang seems wrong

### ISSUE THREAD

**Issue by insanity54** [siw36/ansible-steamcmd#1]
I don't know if this is ansible-steamcmd's fault, but there seems to be an incorrect shbang in /usr/bin/steamcmd--

```
# BEGIN ANSIBLE MANAGED BLOCK
!/bin/bash
```

I think `!/bin/bash` should be `#!/bin/bash`.

steamcmd chokes on this, but seems to run successfully anyway--

```
/usr/bin/steamcmd: line 2: !/bin/bash: No such file or directory
```

**Comment by michaelsstuff** [comment]
It also has to be in the first line

**Comment by siw36** [comment]
Thanks for contributing. I think this happens due to the way ansible templating is handling the hashtag. I’ll open a PR containing the fix this evening.

**Comment by siw36** [comment]
Issues are fixed in #2 

Notify: @insanity54 @michaelsstuff
